src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -128,20 +129,31 @@ public List<String> getVoltageLevelBusbarSectionsIds(UUID networkUuid, String vo
                 .map(BusbarSection::getId).collect(Collectors.toList());
     }
 
-    private List<String> getHvdcLinesIds(UUID networkUuid, String variantId, @NonNull List<String> substationsIds, List<Double> nominalVoltages) {
+    private List<String> getHvdcLinesByHvdcType(UUID networkUuid, String variantId, @NonNull List<String> substationsIds, List<Double> nominalVoltages, HvdcConverterStation.HvdcType type) {
```

Comment:
```suggestion
    private List<String> getHvdcLinesIdsByHvdcType(UUID networkUuid, String variantId, @NonNull List<String> substationsIds, List<Double> nominalVoltages, HvdcConverterStation.HvdcType type) {
```
Just add Ids in the method name